studio/package.json

```diff
@@ -12,6 +12,7 @@
     "@auth0/nextjs-auth0": "^1.6.1",
     "@headlessui/react": "^1.4.1",
     "@monaco-editor/react": "^4.3.1",
+    "@next/bundle-analyzer": "^12.0.4",
```

Comment:
I needed to move this one to a dependency because of the way it's injected in next.config.js - I'm sure there could be a better way to do it, but this gets the job done